### ISSUE TITLE
Pytorch DDP trainning will get stuck at loss.backward() after nvidia driver get updated.

### ISSUE THREAD

**Issue by gemfield** [pytorch/pytorch#44171]
## 🐛 Bug

After Ubuntu host got an automatical update of linux-modules-nvidia-440-5.4.0-45-generic:amd64 driver, the Pytorch DDP trainning in docker container will get stuck at loss.backward(), with cpu 100% and GPU 100% usage.

## To Reproduce

1. Launch Docker container with pytorch/pytorch:1.6.0-cuda10.1-cudnn7-devel
1. Train a model with Pytorch DDP.
1. Will get stuck at first iter, and to be precise, it is loss.backward() in first iter.

I use the class DeepvacDDP in https://github.com/DeepVAC/deepvac/blob/master/deepvac/syszux_deepvac.py to train my model.

## Expected behavior
NA

## Environment
Official pytorch/pytorch:1.6.0-cuda10.1-cudnn7-devel docker image on Ubuntu 20.04 host.

## Additional context

```bash
#######before automatical driver update ##############

root@pytorch160:/home/gemfield# nvidia-smi
......
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 440.64      Driver Version: 440.64      CUDA Version: 10.2     |
......

#######after automatical driver update ##############

root@pytorch160:/home/gemfield# nvidia-smi
......  
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 440.100      Driver Version: 440.100      CUDA Version: 10.2     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  GeForce RTX 208...  Off  | 00000000:19:00.0 Off |                  N/A |
...
```

**Comment by gemfield** [comment]
After the Nvidia driver get updated on Ubuntu 20.04 today, the DDP stuck issue seems not exist any more.
```bash
gemfield@ai03:~$ cat /var/log/apt/history.log | grep -i -C 2 nvidia
Start-Date: 2020-09-03  06:44:01
Commandline: /usr/bin/unattended-upgrade
Install: linux-modules-nvidia-440-5.4.0-45-generic:amd64 (5.4.0-45.49, automatic)
Upgrade: linux-modules-nvidia-440-generic-hwe-20.04:amd64 (5.4.0-42.46, 5.4.0-45.49)
End-Date: 2020-09-03  06:44:33

--
Start-Date: 2020-09-09  06:32:11
Commandline: /usr/bin/unattended-upgrade
Install: linux-modules-nvidia-440-5.4.0-47-generic:amd64 (5.4.0-47.51, automatic), linux-modules-5.4.0-47-generic:amd64 (5.4.0-47.51, automatic), linux-image-5.4.0-47-generic:amd64 (5.4.0-47.51, automatic)
Upgrade: linux-modules-nvidia-440-generic-hwe-20.04:amd64 (5.4.0-45.49, 5.4.0-47.51)
End-Date: 2020-09-09  06:32:32
```
Will close this issue.

**Comment by izdeby** [comment]
Marking as triage review as not sure about this one

**Comment by seemethere** [comment]
Going to close this one since issue seems to be resolved
